mac/expert/install_exercism_via_homebrew_troubleshooting.tw2

```diff
@@ -0,0 +1,12 @@
+::Installing Exercism via Homebrew - Expert - Troubleshooting
```

Comment:
What do you suggest we should do differently for troubleshooting for experts?

Reply:
Cool, would you add that link to this file?

I think we can make it possible to flip back and forth--but I'd like to get a first pass working without it, just to see how complicated it gets before we add more.